recipes/xsdata/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "xsdata" %}
+{% set version = "20.11.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ab484bc3384b9f4e278f245724a3fbe0c868e5551b42b3ba13ead80d498866e1
+
+build:
+  number: 0
```

Comment:
If you drop the selector for `dataclasses` below (as in comment), you can make this recipe `noarch: python`